crates/c/src/lib.rs

```diff
@@ -1091,6 +1085,24 @@ void __wasm_export_{ns}_{snake}_dtor({ns}_{snake}_t* arg) {{
     }
 }
 
+impl<'a> InterfaceGenerator<'a> {
+    fn reosurce_handle_names(&mut self, id: TypeId, name: &str) -> (String, String) {
```

Comment:
Could this be slurped back up into the caller now that it only has one caller again?